tests/plots/test_style.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import asdict
+
+import numpy as np
+
+from shap.plots import _style
+
+
+def test_default_style():
+    new_style = _style.load_default_style()
+    assert configs_are_equal(_style.STYLE, new_style)
+
+    new_style.secondary_color_negative = "black"
+    assert not configs_are_equal(_style.STYLE, new_style)
```

Comment:
Is there much harm in allowing users to mutate StyleConfig classes? This seems like something a user might want to do, and not everyone will be familiar with `dataclasses.replace`.

Are there any downsides in allowing mutation?

Reply:
Having thought about this more, I think you make a good point.

In particular, it's probably not a good design to ask users to replace the _entire_ "Config" object just to change one or two parameters. This would have a few downsides:
- The user has to pass in more information than necessary: all they really need to pass is the changed params
- It becomes more sensitive to breaking changes. E.g. if we add a new style option, then any code that tries to load in a previous StyleConfig will presumably break.

So, I think it would be better to design the API around _overriding_ parameters, rather than having to provide the entire set.

Reply:
Done, the StyleConfig is now frozen.